src/js/interactionevents.tsx

```diff
@@ -0,0 +1,125 @@
+import { addBreadcrumb } from "@sentry/core";
+import { Breadcrumb, Severity } from "@sentry/types";
+import * as React from "react";
+import { StyleSheet, View } from "react-native";
+
+// tslint:disable-next-line: interface-over-type-literal
+export type InteractionEventBoundaryProps = {
+  breadcrumbCategory?: string;
+  elementIdKey?: string;
+};
+
+// tslint:disable-next-line: interface-over-type-literal
+type InteractionEventBoundaryDefaultProps = {
+  breadcrumbCategory: string;
+  elementIdKey: string;
+};
+
+const interactionEventStyles = StyleSheet.create({
+  wrapperView: {
+    flex: 1,
+  },
+});
+
+const DEFAULT_BREADCRUMB_CATEGORY = "interaction";
```

Comment:
@marandaneto Please remind us what the category and type for UI interactions on iOS/Android is to make sure we align this.

Reply:
And do we also need a `type`?

Reply:
answered offline with breadcrumbs docs